src/main/java/com/sap/sgs/phosphor/fosstars/model/score/oss/ProjectActivityScore.java

```diff
@@ -118,7 +152,7 @@ private double commitsPoints(Value<Integer> numberOfCommits) {
    * @param numberOfContributors The number of contributors.
    * @return The sub-score value.
    */
-  private double contributorsPoints(Value<Integer> numberOfContributors) {
+  private static double contributorsPoints(Value<Integer> numberOfContributors) {
```

Comment:
Do we need static here?

Reply:
It doesn't have to but I should since the method doesn't rely on the object's state.

Reply:
ok. The commitsPoints() is not static. Does the same rule apply here as well. I mean should it be static?

Reply:
Yes, you're absolutely right. `commitsPoints()` should also be static. I'll update it.